webpack/assets/javascripts/react_app/components/SearchBar/search-bar.scss

```diff
@@ -0,0 +1,41 @@
+@import "../AutoComplete/auto-complete.scss";
+
+.search-bar {
+  .autocomplete-aux {
+    right: 108px;
+    // Keeps in position when screen size is xs and search button shrinks.
+    @media screen and (max-width: 767px) {
```

Comment:
AFAIK you can use classes instead `@media` , such as `visible-xs-block`

Reply:
They do exactly the same though `visible-xs-block` rely on bootstrap

Reply:
so if we have clasess that already implements that `@media` query, shouldn't we use it?

Reply:
Another thing is that we use a class which is generated by the Autocomplete component which is used also by other components that doesn't need this behaviour

Reply:
IMO it is better to use the patternfly/bootstrap variables so we can reuse the same values all over the project and change it easily.
See: https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.scss#L23